web/src/components/expenses/EstimateCreateDialog.vue

```diff
@@ -83,80 +83,80 @@
             Save
           </v-btn>
         </v-card-actions>
+        {{ departureDate }}
+        {{ returnDate }}
```

Comment:
Debugging code (that should be removed)?

Reply:
Removed in https://github.com/icefoganalytics/travel-authorization/pull/324/commits/c487efa61309d209b9a95f5d359111e9a06e2816